modules/apps/jaggery/emm/src/emm/modules/policy.js

```diff
@@ -85,10 +85,10 @@ policyModule = function () {
                 response.status = "success";
                 response.content = policyListToView;
                 return response;
-            },function(responsePayload){
+            }, function (responsePayload) {
```

Comment:
better placing the error callback as a separate function and call it here for readability